app/scripts/filters/resources.js

```diff
@@ -1322,8 +1322,8 @@ angular.module('openshiftConsole')
       return cmd;
     };
 
-    return function(container, image) {
-      if (!container || !image) {
+    return function(container, image, returnArgsIfNoCmd) {
```

Comment:
Should `returnArgsIfNoCmd` just always be true? We should update debug terminal to set that, and that's the only other place the filter is used..